tests/php/test-amp-style-sanitizer.php

```diff
@@ -25,6 +25,8 @@ class AMP_Style_Sanitizer_Test extends WP_UnitTestCase {
 	use MarkupComparison;
 	use PrivateAccess;
 
+	private $original_theme_directories;
```

Comment:
I suppose this is to clean up any stuff the tests may have altered.

Reply:
That's right.